src/functionaltests/java/com/ericsson/ei/utils/FunctionalTestBase.java

```diff
@@ -182,40 +175,36 @@ protected JsonNode getJSONFromFile(String filePath) throws IOException {
 
     /**
      * Verify that aggregated object contains the expected information.
-     * 
-     * @param arguments
-     *            list of arguments to check
-     * @return list of missing arguments
+     *
+     * @param checklist list of checklist to check
+     * @return list of missing checklist
      * @throws InterruptedException
      */
-    protected List<String> verifyAggregatedObjectInDB(List<String> arguments) throws InterruptedException {
-        List<String> checklist = new ArrayList<String>(arguments);
-        MongoClient mongoClient = new MongoClient(getMongoDbHost(), getMongoDbPort());
+    protected List<String> verifyAggregatedObjectInDB(List<String> checklist) throws InterruptedException {
         long stopTime = System.currentTimeMillis() + 30000;
         while (!checklist.isEmpty() && stopTime > System.currentTimeMillis()) {
             checklist = compareArgumentsWithAggregatedObjectInDB(checklist);
             if (checklist.isEmpty()) {
                 break;
             }
-            TimeUnit.MILLISECONDS.sleep(3000);
+            TimeUnit.MILLISECONDS.sleep(1000);
         }
-        mongoClient.close();
         return checklist;
     }
 
     /**
      * Checks that aggregated object contains specified arguments.
-     * 
-     * @param checklist
-     *            list of arguments
+     *
+     * @param checklist list of arguments
      * @return list of missing arguments
      */
     private List<String> compareArgumentsWithAggregatedObjectInDB(List<String> checklist) {
+        mongoClient = new MongoClient(getMongoDbHost(), getMongoDbPort());
         MongoDatabase db = mongoClient.getDatabase(database);
         MongoCollection<Document> table = db.getCollection(aggregatedCollectionName);
-        List<Document> documents = table.find().into(new ArrayList<Document>());
+        List<Document> documents = table.find().into(new ArrayList<>());
         for (Document document : documents) {
-            for (String expectedValue : new ArrayList<String>(checklist)) {
+            for (String expectedValue : new ArrayList<>(checklist)) {
```

Comment:
rename checklist to same name as other places.

Reply:
it's not a name this a size for arraylist

Reply:
I mean the name of the variable. You renamed "checklist" to "eventsIdList" earlier in the code, you should use same name. Or is "checklist" something different here?

Reply:
in this place checklist is size for arraylist and here not any code related to eventsIdList

Reply:
Sorry my mistake it not a size and we can simplify this line but still nothing related to eventIdList